core/src/test/java/org/ocpsoft/PrettyTimeI18n_SO_Test.java

```diff
@@ -0,0 +1,508 @@
+/*
+ * Copyright 2012 <a href="mailto:lincolnbaxter@gmail.com">Lincoln Baxter, III</a>
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ocpsoft.prettytime;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.ocpsoft.prettytime.format.SimpleTimeFormat;
+import org.ocpsoft.prettytime.impl.ResourcesTimeFormat;
+import org.ocpsoft.prettytime.units.Minute;
+
+public class PrettyTimeTest
+{
+   private Locale locale;
+   private LocalDateTime now;
+
+   @Before
+   public void setUp() throws Exception
+   {
+      locale = Locale.getDefault();
+      Locale.setDefault(Locale.ROOT);
```

Comment:
```suggestion
      Locale.setDefault(new Locale("so","SO"));
```

Reply:
Agreed. This probably needs to be set to the SO locale since the tests would fail on non-SO environments.

Reply:
Good catch.